libr/bin/format/mach0/mach0.c

```diff
@@ -2465,7 +2465,7 @@ static char *get_name(struct MACH0_(obj_t) *mo, ut32 stridx, bool filter) {
 }
 
 static int walk_exports(struct MACH0_(obj_t) *bin, RExportsIterator iterator, void * ctx) {
-#define ULEB(at) read_uleb128 (&p, end)
+#define ULEB() read_uleb128 (&p, end)
```

Comment:
Uhm.. is this macro used at all? Because you changed the signature but the build didnt break

Reply:
Yes it's used at https://github.com/radareorg/radare2/blob/master/libr/bin/format/mach0/mach0.c#L2509 and afterwards. But what surprised me is the original version was compiled successfully.

Reply:
interesting. anyway. thanks!